<ui version="4.0" >
 <class>FilterFitTab</class>
 <widget class="QWidget" name="FilterFitTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>338</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="GroupBox9_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Plugin Selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QComboBox" name="_pluginCombo" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Plugin Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="_pluginLabel" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="_descriptionLabel" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_inputOutputBox" >
     <property name="title" >
      <string>Inputs and Outputs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Kst::CurveAppearance" native="1" name="_curveAppearance" />
   </item>
  </layout>
  <zorder>GroupBox9_3</zorder>
  <zorder>_inputOutputBox</zorder>
  <zorder>_curveAppearance</zorder>
  <zorder>_curveAppearance</zorder>
  <zorder>label</zorder>
  <zorder>_pluginLabel</zorder>
  <zorder>label</zorder>
  <zorder>_pluginLabel</zorder>
  <zorder>label_3</zorder>
  <zorder>_descriptionLabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Kst::CurveAppearance</class>
   <extends>QWidget</extends>
   <header>curveappearance.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_pluginCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
